### ISSUE TITLE
Can you design a bigger model like smt_huge?

### ISSUE THREAD

**Issue by Geek-lixiang** [AFeng-x/SMT#17]
Now the biggest model is smt_large, Can you design a bigger model like smt_huge?
Thank you very much!

**Comment by AFeng-x** [comment]
Yes, it's feasible from a design perspective. However, due to the limited availability of GPU machines at the moment, we may temporarily be unable to complete the training of larger models. Nevertheless, this is an experiment we plan to continue in the future. Thank you.
